strongbox-user-management/pom.xml

```diff
@@ -252,6 +252,39 @@
         </dependency>
 
         <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-ldap</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-config</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.directory.server</groupId>
+            <artifactId>apacheds-core</artifactId>
+            <version>${version.apache.directory}</version>
```

Comment:
You can drop the `<version/>` part here, as it's already inherited from the parent `pom.xml`.